includes/class-wc-checkout.php

```diff
@@ -980,6 +980,8 @@ protected function process_order_payment( $order_id, $payment_method ) {
 			}
 
 			wp_send_json( $result );
+		}else{
```

Comment:
Need to tweak the formatting here.

```suggestion
		} else {
```